leo/doc/LeoDocs.leo

```diff
@@ -9483,7 +9482,7 @@ Where '%s' is replaced with the full pathname.
 
 **Note**: This plugin terminates handling of the 'icondclick1' event by returning True. If another plugin using this event (e.g. vim.py) is also enabled, the order in ``@enabled-plugins`` matters. For example: if vim.py is enabled before mime.py, double-clicking on an ``@mime`` node will both open the body text in [g]vim AND call the mime_open_cmd.
 
-This plugin is complementary to the UNL.py plugin's ``@url`` nodes. Use ``@url`` for opening either URLs or Uniform Node Locators in "\*.leo" files and use ``@mime`` nodes for opening files on the local file system. It also replaces the startfile.py plugin, where here the headline must start with ``@mime`` to activate this plugin.
+Use ``@url`` for opening either URLs or Uniform Node Locators in "\*.leo" files and use ``@mime`` nodes for opening files on the local file system. It also replaces the startfile.py plugin, where here the headline must start with ``@mime`` to activate this plugin.
```

Comment:
The second sentence should be removed also. The one that begins with "It also replaces... "